packages/config/src/forkConfig/types.ts

```diff
@@ -41,4 +41,6 @@ export type ForkConfig = {
   getBlobsForkTypes(slot: Slot): SSZTypesFor<ForkBlobs>;
   /** Get max blobs per block by hard-fork */
   getMaxBlobsPerBlock(fork: ForkName): number;
+  /** Get max request blob sidecars by hard-fork */
+  getMaxRequestBlobSidecars(fork: ForkName): number;
```

Comment:
Not a huge fan of the explosion of methods here, but what can you do...

something like?

```ts
type ConfigValue {
  MAX_BLOBS_PER_BLOCK: number;
  MAX_REQUEST_BLOB_SIDECAR: number;
}

...

  getValue<K extends keyof ConfigValue>(fork: ForkName, name: K): ConfigValue[K] {
    switch (name) {
      case "MAX_BLOBS_PER_BLOCK": return isForkPostElectra(fork)
        ? config.MAX_BLOBS_PER_BLOCK_ELECTRA
        : config.MAX_BLOBS_PER_BLOCK;
      case "MAX_REQUEST_BLOB_SIDECAR": return isForkPostElectra(fork)
        ? config.MAX_REQUEST_BLOB_SIDECARS_ELECTRA
        : config.MAX_REQUEST_BLOB_SIDECARS;
    }
  }
```

Reply:
I think it's a good idea :+1: